### PR TITLE
Add desktop-version

### PR DESCRIPTION
**Pull request by ecekpo**
I have added changes to Portfolio: desktop-version.

### CODE REVIEW

**Review by Kandy-Peter — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @ecekpo  
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has some issues needed to be fixed before reviewing it:
- [ ] Make sure to give your PR a meaningful description of what you implemented in this milestone, so it will look professional as it is described in this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

- [ ] Ensure that all LInters are greens(i.e No linter that failed).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -4,263 +4,310 @@
...
-                    <input type="text" name="username" id="name" placeholder="Your name" maxlength="30" required/>
-                   
-                    <input type="email" name="email" id="email" placeholder="Your email" required/>
+                <input type="text" name="username"  maxlength="30" placeholder="Full name" required>
+
+                <input type="email" name="email" placeholder="Email" required>
 
                 <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
-                <button class="form-submit" type="submit">Get In Touch </button>
+                <button type="submit" class="click" value="Submit">Get In Touch</button>
```
Comment (ishimwezachee) [inline index.html:308]: - [ ] You've done an incredible job to apply the requirement of the project, but you also have to match your design to the one provided from figma design.  Your buttons seemed to be position in wrong place. please fix it to match to the provided design. 
image

image

File: index.html [hunk 2]
```diff
@@ -4,263 +4,310 @@
...
-              <p class="sub1">CANOPY</p>
-              <div class="circle"></div>
-              <p class="sub2">Back End Dev</p>
-              <div class="circle"></div>
-              <p class="sub2">2015</p>
-          </div>
+    <header>
+      <nav class="nav-bar">
+          <div class="logonav">
+            <a href="#">My Logo</a>
```
Comment (ishimwezachee) [inline index.html:19]: - [ ]  You should add the margin similar to the one provided from Figma design. 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @ecekpo   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

**Review by elyor-doniyorov — APPROVED** [review]
Hi @ecekpo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ There are no linter errors
✔️ Your design matches the Figma design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**